modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -201,3 +201,279 @@
+
+/obj/item/ammo_casing/security_pistol/influence
+	name = "influence security pistol casing"
+	projectile_type = /obj/projectile/bullet/security_pistol/influence
+	harmful = FALSE
+
+/obj/projectile/bullet/security_pistol/influence
+	name = "influence security pistol bullet"
+	damage = 41
+	damage_type = STAMINA
+	ricochets_max = 6
+	ricochet_incidence_leeway = 0
+	ricochet_chance = 130
+	ricochet_decay_damage = 0.8
+	wound_bonus = CANT_WOUND
+
+// TRAC rounds. Lower raw damage. Puts a tracking chip in the target, embeds. Penetrates armor.
+/obj/item/ammo_box/magazine/security_pistol/trac
+	name = "trac security pistol magazine"
+	desc = "A magazine for the security pistol. Loaded with TRAC rounds, which embed a tracking chip in the target, while also being handy against armor."
+	icon_state = "security_pistol_trac"
+	base_icon_state = "security_pistol_trac"
+	ammo_type = /obj/item/ammo_casing/security_pistol/trac
+
+/obj/item/ammo_casing/security_pistol/trac
+	name = "trac security pistol casing"
+	projectile_type = /obj/projectile/bullet/security_pistol/trac
+
+/obj/projectile/bullet/security_pistol/trac
+	name = "trac security pistol bullet"
+	damage = 25
+	armour_penetration = 75
+	embedding = list(embed_chance=75, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=1 SECONDS)
+	wound_falloff_tile = -5
+	embed_falloff_tile = -15
+
+/obj/projectile/bullet/security_pistol/trac/on_hit(atom/target, blocked = FALSE)
+	. = ..()
+	var/mob/living/carbon/M = target
+	if(!istype(M))
+		return
+	var/obj/item/implant/tracking/c38/imp
+	for(var/obj/item/implant/tracking/c38/TI in M.implants) //checks if the target already contains a tracking implant
+		imp = TI
+		return
+	if(!imp)
+		imp = new /obj/item/implant/tracking/c38(M)
+		imp.implant(M)
+
+// Iceblox Rounds. Lower raw damage. Makes the target cold.
+/obj/item/ammo_box/magazine/security_pistol/iceblox
+	name = "iceblox security pistol magazine"
+	desc = "A magazine for the security pistol. Loaded with Iceblox rounds, which make the target cold."
+	icon_state = "security_pistol_iceblox"
+	base_icon_state = "security_pistol_iceblox"
+	ammo_type = /obj/item/ammo_casing/security_pistol/iceblox
+
+/obj/item/ammo_casing/security_pistol/iceblox
+	name = "iceblox security pistol casing"
+	projectile_type = /obj/projectile/bullet/security_pistol/iceblox
+
+/obj/projectile/bullet/security_pistol/iceblox
+	name = "iceblox security pistol bullet"
+	damage = 25
+	var/temperature = 100
+
+/obj/projectile/bullet/security_pistol/iceblox/on_hit(atom/target, blocked = FALSE)
+	. = ..()
+	if(isliving(target))
+		var/mob/living/M = target
+		M.adjust_bodytemperature(((100-blocked)/100)*(temperature - M.bodytemperature))
+
+// Cruelty rounds. Lower raw damage. Aggressively violent wounds and embeds.
+/obj/item/ammo_box/magazine/security_pistol/cruelty
+	name = "cruelty security pistol magazine"
```

Comment:
Again, hollowpoint.

Reply:
I'm sticking to the names I've got for the ammo types, but I'll make sure the magazine descriptions clarify things.